src/main/java/seedu/realodex/model/remark/Remark.java

```diff
@@ -15,9 +17,21 @@ public class Remark {
      * @param remarkName A valid remark.
      */
     public Remark(String remarkName) {
+        requireNonNull(remarkName);
         this.remarkName = remarkName;
     }
 
+    /**
+     * Returns true if a given string is a valid name.
+     * However, as of v1.2 there is no concrete check; hence it will always return true.
```

Comment:
is this even a valid javadoc

Reply:
🤔

Reply:
actually what would be a "invalid remark"